src/quilla/__init__.py

```diff
@@ -96,30 +181,19 @@ def execute(ctx: Context, json_data: str) -> int:
         there were any reports that were flagged as failed
     '''
 
-    ui_validation = UIValidation.from_json(ctx, json_data)
```

Comment:
Replace all UIValidation with Quilla?

Reply:
I originally figured that leaving it as UIValidation made sense, since at its core a Quilla test is an UI validation, but I think you're right. I'm going to rename it `QuillaTest` to better represent that